app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(17, 4)) == 17
```

Comment:
Here, you are performing an operation (sum) on the result of the function before comparing it with the expected result. It would be better to directly compare the function result with the expected result. This way, it's easier to understand what the function is expected to return.